pkgs/desktops/gnome-3/extensions/dynamic-panel-transparency/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, gnome3, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-dynamic-panel-transparency";
+  version = "35";
+
+  src = fetchFromGitHub {
+    owner = "ewlsh";
+    repo = "dynamic-panel-transparency";
+    rev = "0800c0a921bb25f51f6a5ca2e6981b1669a69aec";
```

Comment:
https://github.com/ewlsh/dynamic-panel-transparency/issues/135

Reply:
The [commit](https://github.com/ewlsh/dynamic-panel-transparency/commits/master/dynamic-panel-transparency%40rockon999.github.io/metadata.json) i've picked is the last one that touched the `metadata.json`

Thank you for opening the PR.